code/game/machinery/frame.dm

```diff
@@ -180,9 +175,6 @@
 	x_offset = 24
 	y_offset = 24
 
-/datum/frame/frame_types/cancel //used to get out of input dialogue
-	name = "Cancel"
```

Comment:
Oh wow, they didn't want to just add "Cancel" to the list, which is what I did in vorepanel before I knew better. this is great.